docs/installation/cloud-ex-machine-ocean.md

```diff
@@ -0,0 +1,239 @@
+
+        * Documentation:  https://help.ubuntu.com/
+
+        System information as of Mon Dec 21 21:38:53 EST 2015
+
+        System load:  0.77               Processes:              70
+        Usage of /:   11.4% of 19.56GB   Users logged in:        0
+        Memory usage: 15%                IP address for eth0:    45.55.139.48
+        Swap usage:   0%                 IP address for docker0: 172.17.0.1
+
+        Graph this data and manage this system at:
+        https://landscape.canonical.com/
+
+7. Verify Docker Engine is installed correctly by running Hello World.
+
+          ubuntu@ip-xxx-xx-x-xxx:~$ sudo docker run hello-world
+          ubuntu@ip-172-31-0-151:~$ sudo docker run hello-world
+          Unable to find image 'hello-world:latest' locally
+          latest: Pulling from library/hello-world
+          b901d36b6f2f: Pull complete
+          0a6ba66e537a: Pull complete
+          Digest: sha256:8be990ef2aeb16dbcb9271ddfe2610fa6658d13f6dfb8bc72074cc1ca36966a7
+          Status: Downloaded newer image for hello-world:latest
+
+          Hello from Docker.
+          This message shows that your installation appears to be working correctly.
+
+          To generate this message, Docker took the following steps:
+          1. The Docker client contacted the Docker daemon.
+          2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+          3. The Docker daemon created a new container from that image which runs the executable that produces the output you are currently reading.
+          4. The Docker daemon streamed that output to the Docker client, which sent it to your terminal.
+
+          To try something more ambitious, you can run an Ubuntu container with:
+          $ docker run -it ubuntu bash
+
+          Share images, automate workflows, and more with a free Docker Hub account:
+          https://hub.docker.com
+
+          For more examples and ideas, visit:
+          https://docs.docker.com/userguide/
+
+      You can type keyboard command Control-D or `exit` to log out of the remote server.
+
+#### Understand the defaults and options on the create command
+
+For convenience, `docker-machine` will use sensible defaults for choosing settings such as the image that the server is based on, but you override the defaults using the respective flags (e.g. `--digitalocean-image`). This is useful if, for example, you want to create a cloud server with a lot of memory and CPUs (by default `docker-machine` creates a small server). For a full list of the flags/settings available and their defaults, see the output of `docker-machine create -h` at the command line. See also <a href=""https://docs.docker.com/machine/drivers/os-base/" target="_blank">Driver options and operating system defaults</a> and information about the <a href="https://docs.docker.com/machine/reference/create/ target="_blank">`create`</a> command in the Docker Machine documentation.
```

Comment:
Wrap...
Also "but you can override" ? 
This is > Overrides are maybe?
I might break this chunky para at the sentence For.